components/atom/checkbox/src/index.scss

```diff
@@ -21,6 +21,8 @@ $bdrs-atom-checkbox: $bdrs-m !default;
 $c-atom-checkbox: $c-white !default;
 $h-atom-checkbox: 24px !default;
 $w-atom-checkbox: 24px !default;
+$h-atom-checkbox-icon: 16px !default;
+$w-atom-checkbox-icon: 16px !default;
```

Comment:
what do you think about using sui-theme icon sizes? so we can easily keep consistency between components in further iterations.

https://github.com/SUI-Components/sui-theme/blob/cf07c47c68541bdb8a2f550235e96fe52bd34d83/src/settings/_size.scss#L7